upgrade/capsule.py

```diff
@@ -57,6 +60,10 @@ def satellite6_capsule_setup(sat_host, os_version, upgradable_capsule=True):
         execute(delete_rhevm_instance, cap_instance)
         logger.info('Turning on Capsule Instance ....')
         execute(create_rhevm_instance, cap_instance, cap_image)
+        for cap_host in cap_hosts:
+            if not host_pings(cap_host):
+                sys.exit(1)
```

Comment:
You are not yet fixed the checking of all capsules in for loop.
Create one list and append with names of non pinging capsules and then print that list.
Then exit the code.